.circleci/config.yml

```diff
@@ -79,7 +99,17 @@ commands:
               --var AUTH_CLIENT_ID=${<< parameters.auth_client_id >>} \
               --var AUTH_CLIENT_SECRET=${<< parameters.auth_client_secret >>} \
               --var NEW_RELIC_LICENSE_KEY=${<< parameters.new_relic_license >>} \
+              --var POSTGRES_DB=${<< parameters.postgres_db >>} \
+              --var POSTGRES_HOST=${<< parameters.postgres_host >>} \
+              --var POSTGRES_PASSWORD=${<< parameters.postgres_password >>} \
+              --var POSTGRES_USERNAME=${<< parameters.postgres_username >>} \
               --var SESSION_SECRET=${<< parameters.session_secret >>}
+      - run:
+          name: Run database migrations
+          command: |
+            cf run-task << parameters.app_name >> "yarn db:migrate:prod"
+            sleep 20
+            cf restart << parameters.app_name >>
```

Comment:
Why do we need to sleep and restart?

Reply:
You know, this is a fair question. I copied this from the original demo app config. According to the [docs](https://docs.cloudfoundry.org/devguide/deploy-apps/start-restart-restage.html#restart) 
> Restart your app to refresh the app’s environment after actions such as binding a new service to the app or setting an environment variable that only the app consumes

We shouldn't be updating any vars here or binding a new service. I think we can just use `cf run-task` and be done. 
@rahearn Do you have any insight as to why you restarted the app? https://github.com/18F/Head-Start-TTA/blob/main/.circleci/config.yml#L143

Reply:
A rails app reads the db schema on startup to build its data model, so changing the data model out from under it can result in runtime issues.

This was the sledgehammer approach to ensuring that the running app is referencing the correct schema.

Reply:
Okay, I took out the `sleep` and `restart` steps and from watching the log stream from the newly pushed application everything seems to work fine.